tests/tabular_output/test_tabulate_adapter.py

```diff
@@ -19,3 +19,20 @@ def test_tabulate_wrapper():
         | abc       |        1 |
         | d         |      456 |
         +-----------+----------+''')
+
+
+def test_markup_format():
+    """Test that markup formats do not have number align or string align."""
```

Comment:
@meeuw, Ah, I see the problem, it was a simple typo -- it should be fixed in this PR now.